Revisit the "Developer Framework" getting started section (with bonus tests section rework) (#169)
validmind/documentation: site/guide/get-started-developer-framework.qmd

---
title: "Get started with the ValidMind Developer Framework"
date: last-modified
listing:
  - id: developer-getting-started
    type: grid
    max-description-length: 250
    sort: false
    fields: [title, description]
    contents:
    - ../notebooks/quickstart_customer_churn_full_suite.ipynb
    - developer-getting-started-video.qmd
  - id: developer-how-to-beginner
    type: grid
    max-description-length: 250
    sort: false
    fields: [title, description]
    contents: 
    - ../notebooks/how_to/run_a_test.ipynb
    - ../notebooks/how_to/configure_parameters_demo.ipynb
    - ../notebooks/how_to/implementing_custom_tests.ipynb
  - id: developer-code-samples
    type: grid
    max-description-length: 250
    sort: false
    fields: [title, description]
    contents: 
    - ../notebooks/code_samples/customization/customizing_metrics_with_output_templates.ipynb
    - ../notebooks/code_samples/LLM_and_NLP/prompt_validation_demo.ipynb
    - ../notebooks/code_samples/time_series/tutorial_time_series_forecasting.ipynb

---

> As a model developer, you use the ValidMind Developer Framework to automate the generation of model documentation by uploading documentation artifacts and test results to the ValidMind platform.

Reviewer: ```suggestion
The ValidMind Developer Framework helps you streamline model documentation by automating the generation of drafts. All you need to do is upload your documentation artifacts and test results to the ValidMind platform.
```

- **Generate documentation**: Through automation, the framework extracts metadata from associated datasets and models for you and generates model documentation based on a template. You can also add more documentation and tests manually using the documentation editing capabilities in the ValidMind UI. 

- **Run validation tests**: The framework provides a suite of validation tests for common financial services use cases. For cases where these tests do not cover everything you need, you can also extend existing test suites with your own proprietary tests or testing providers. 

The Developer Framework is designed to be model agnostic. If your model is built in Python, ValidMind’s Python client library already provides all the standard functionality you might need without requiring your developers to rewrite any functions.

::: {.callout-important collapse="true" appearance="minimal"}
##  {{< fa angle-right >}} Key ValidMind concepts

<!--- These concepts were already introduced in the "Get started" section. They are repeated here in a collapsed format in case a reader missed them earlier. --->

{{< include key-validmind-concepts.qmd >}}

:::

## Getting started

> After you [**sign up**](https://app.prod.validmind.ai) for ValidMind to get access, start with our QuickStart or watch the video:

Reviewer: ```suggestion
After you [**sign up**](https://app.prod.validmind.ai) for ValidMind to get access, head to the quickstart guide or watch the video demonstration:
```

Author: I edited this to change back to "QuickStart" — we should change this globally if that's the new spelling — and committed the rest.
